models/models.js

```diff
@@ -17,3 +18,16 @@ exports.selectArticles = (sort_by = "created_at", order = "desc") => {
     })
 };
 
+exports.selectArticlebyID = (article_id) => {
+    return checkArticleExists(article_id).then(()=>{
```

Comment:
`checkArticleExists` is a good implementation however not exactly necessary for this endpoint - it looks like you're simply repeating the exact same behaviour. This example was more appropriate for getting comments for an article that DOES exist, but has no comments associated.
I would remove its use here.